### ISSUE TITLE
Inline code is displayed as block with Coy theme

### ISSUE THREAD

**Issue by uranusjr** [PrismJS/prism#675]
As can be seen in the “Remove initial line feed” plugin page. With the default theme:

![screen shot 2015-08-17 at 16 29 15](https://cloud.githubusercontent.com/assets/605277/9300911/1fdf8f48-44fd-11e5-9b5d-7daf997bf3a1.png)

With Coy:

![screen shot 2015-08-17 at 16 29 41](https://cloud.githubusercontent.com/assets/605277/9300958/6bac12b6-44fd-11e5-872e-f722b226b75c.png)

**Comment by Golmote** [comment]
@tshedor?

**Comment by AnujRNair** [comment]
I ran into the same issue today.
A quick fix is available by adding the following CSS code:

```css
:not(pre) > code[class*="language-"] {
    display: inline;
}
```

**Comment by Golmote** [comment]
Fixed by #725.
